test/parallel/test-process-exit-flaky-handler.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+require('../common');
+
+// This test ensures that the 'exit' event handler is not flaky, and
```

Comment:
Tiny nit: I'd simplify like "This test ensures that no asynchronous operations are performed in the `'exit'` event handler".